test.sh

```diff
@@ -11,6 +11,11 @@ mkdir -p test
 # This is a temporary workaround until we properly implement volumes
 chcon -R -t container_file_t email
 
+command -v curl || { echo "error: curl command required yet absent" ; exit 1 ; }
+command -v flake8 || { echo "error: flake8 command required yet absent" ; exit 1 ; }
+command -v chcon || { echo "error: chcon command required yet absent" ; exit 1 ; }
+command -v podman || { echo "error: podman command required yet absent" ; exit 1 ; }
+
```

Comment:
Also, maybe this commit could / should come before the one before since in this order you could experience weird error messages if you happened to be bisecting and didn't have your dev env setup.

I also think that the spelling fixes could come first overall.